llvm/test/tools/llvm-objcopy/ELF/add-invalid-note.test

```diff
@@ -20,6 +20,12 @@
 # RUN: printf "\x08\x00\x00\x00" >  %t-short-note.bin
 # RUN: printf "\x07\x00\x00\x00" >> %t-short-note.bin
 
+## Test compatibility with .note.gnu.property
```

Comment:
Nit: I'd personally move this block down to immediately above the individual case that uses it. This comment should probably also mention something about the 8-byte alignment that is the "interesting" feature of it.